src/client/css/play.css

```diff
@@ -985,8 +985,28 @@ button.join-button, button.copy-button {
     box-shadow: 0px 0px 9px 0px rgba(0, 0, 0, 0.259);
 }
 
+.dotred {
+    background-color: red;
+    box-shadow: 0px 0px 9px 0px rgba(0, 0, 0, 0.491);
+}
+
+.dotgreen {
+    background-color: green;
+    box-shadow: 0px 0px 9px 0px rgba(0, 0, 0, 0.491);
+}
```

Comment:
Hmm. Maybe we should instead use javascript to edit the styling of the dot, so that we don't have to create a new css selector every time we add a new color.